config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  post '/movies/:id/add_actor', to: 'movies#add_actor'
```

Comment:
I mentioned this route being incorrect earlier but again this route isn't RESTful. We want the verb+path to be: `post "/movies/:id/actors", to: "actor_movies#create"`